### PR TITLE
Various smaller fixes

### PR DESCRIPTION
**Pull request by MyIgel**
can be merged at any time ;)

### CODE REVIEW

**Comment by weeman1337** [comment]
> can be merged at any time ;)

:P 

image

**Comment by MyIgel** [comment]
@weeman1337 That is now fixed too ;)

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -131,12 +131,15 @@ vendor/bin/phpunit --testsuite Feature
 vendor/bin/phpunit
 ```
 
-To run code coverage reports its highly recommended to use [`pcov`](https://github.com/krakjoe/pcov) or
+To run code coverage reports it's highly recommended to use [`pcov`](https://github.com/krakjoe/pcov) or
 at least `phpdbg -qrr`(which has problems with switch case statements) as using Xdebug slows down execution.
 ```bash
-php -d pcov.enabled=1 -d pcov.directory=. vendor/bin/phpunit --testsuite Unit --coverage-text
+php -d pcov.enabled=1 -d pcov.directory=. vendor/bin/phpunit --coverage-text
```
Comment (weeman1337) [inline DEVELOPMENT.md:137]: I would mention `docker exec -ti engelsystem_dev-es_workspace-1 composer phpunit:coverage` here as well.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -148,7 +151,7 @@ A browsable HTML version is available at http://localhost:5080/coverage/index.ht
 
 ### Var Dump server
 Symfony Var Dump server is configured to allow for easier debugging. It is not meant as a replacement for xdebug but can actually be used together with xdebug.
-This Var Dump Server is especially useful for when you want to debug a request without messing up the output e.g API calls ot HTML layout.
+This Var Dump Server is especially useful when you want to debug a request without messing up the output e.g. on API calls or the HTML layout.
```
Comment (weeman1337) [inline DEVELOPMENT.md:154]:
```suggestion
This Var Dump Server is especially useful when you want to debug a request without messing up the output e.g. of API calls or the HTML layout.
```

:upside_down_face:

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**